plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Browse.js

```diff
@@ -166,6 +171,16 @@ Ext.define('NX.coreui.controller.Browse', {
   },
 
   /**
+   * @override
+   * Load all of the stores associated with this controller
+   */
+  loadStores: function() {
+    var me = this;
```

Comment:
IIRC single uses of the `me` variable were to be avoided in place of using `this` directly? ie. replace this section with `this.getStore('Repository').load();`